core/frontend/src/TwoWayViewportSync.ts

```diff
@@ -15,28 +15,38 @@ import { Viewport } from "./Viewport";
  * @public
  */
 export class TwoWayViewportSync {
-  private readonly _disconnect: VoidFunction[] = [];
+  protected readonly _disconnect: VoidFunction[] = [];
   private _isEcho = false;
 
   private syncView(source: Viewport, target: Viewport) {
-    if (this._isEcho) return;
+    if (this._isEcho)
+      return;
+
     this._isEcho = true; // so we don't react to the echo of this sync
-    target.applyViewState(source.view.clone(target.iModel));
+    this.syncViewports(source, target);
     this._isEcho = false;
   }
 
+  protected syncViewports(source: Viewport, target: Viewport): void {
+    target.applyViewState(source.view.clone(target.iModel));
+  }
+
+  protected connectViewports(source: Viewport, target: Viewport): void {
+    const viewState = source.view.clone(target.iModel);
+    target.applyViewState(viewState);
+  }
+
   /** Establish the connection between two Viewports. When this method is called, `view2` is initialized with the state of `view1`.
    * Thereafter, any change to the frustum of either view will be reflected in the frustum of the other view.
    */
-  public connect(view1: Viewport, view2: Viewport) {
+  public connect(source: Viewport, target: Viewport) {
```

Comment:
i don't understand the change to "source/target" terminology here, when the connection is two way other than the initialization. I think the explanation in the comment suffices, although it should not say "frustum". The doc for the class should explain better that it synchronizes more than just the frustum.

Reply:
Addressed.